### PR TITLE
Add Ledger & Withdrawals services

### PR DESCRIPTION
**Pull request by markkkkas**
Ledger API:
- [x] Resources
- [x] Service
- [x] Tests 
- [x] Documentation

Withdrawals API:
- [x] Resources
- [x] Service
- [x] Tests 
- [x] Documentation

I've also added new exceptions and refactored some imports, redundant classes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,6 +241,128 @@ PaginatedRefunds(
 )
 ```
 
+## Ledger API
+
+### Get Account
+Retrieves a specific ledger account. This is private API endpoint and requires authentication.
+
+```py
+>>> client.ledger.get()
```
Comment (vgumonis) [inline README.md:250]: id missing

File: README.md [hunk 2]
```diff
@@ -241,6 +241,128 @@ PaginatedRefunds(
...
+        )
+    ]
+)
+```
+
+### Get Withdrawal
+Retrieves withdrawal by ID. This is private API endpoint and requires authentication.
+
+```py
+>>> client.withdrawal.get_all()
```
Comment (vgumonis) [inline README.md:341]: wrong method name

File: coingate/resources/withdrawal.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+from decimal import Decimal
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+from .platform import Platform
+from .public import Currency
+
+
+class PayoutSetting(BaseModel):
```
Comment (vgumonis) [inline coingate/resources/withdrawal.py:11]: would it be possible not to strongly type PayoutSetting ? as it can wan have any variations

**Review by vgumonis — CHANGES_REQUESTED** [review]

File: coingate/resources/withdrawal.py (hunk 3 above)
Comment (markkkkas) [reply coingate/resources/withdrawal.py:11]: Sure. Updated to `Dict[str, Any]`

**Review by vgumonis — APPROVED** [review]

**Pull request merged**